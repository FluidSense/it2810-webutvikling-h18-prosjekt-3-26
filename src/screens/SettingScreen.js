import React from 'react';
import { View, Text, Button, } from 'react-native';

import PedometerSettings from './../components/Pedometer/PedometerSettings.js'


class SettingScreen extends React.Component {
    render() {
      console.log("Settings props",this.props);
      return (
        <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center', backgroundColor: 'green', }}>
          <Text>Settings</Text>
          <Text>Register your pedometer</Text>
          <PedometerSettings startDate={this.props.startDate} updateAvailability={this.props.updateAvailability} updateGlobalSteps={this.props.updateGlobalSteps}/>
        </View>
      );
    }
  }

export default SettingScreen;
