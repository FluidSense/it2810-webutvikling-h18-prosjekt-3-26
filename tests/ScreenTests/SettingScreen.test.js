import React from 'react';
import SettingScreen from '../../src/screens/SettingScreen';

import renderer from 'react-test-renderer';


test('Renders correctly', () => {
  const tree = renderer.create(<SettingScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});