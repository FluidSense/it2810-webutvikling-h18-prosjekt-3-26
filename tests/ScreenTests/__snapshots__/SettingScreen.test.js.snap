// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 30,
      }
    }
  >
    Settings
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Register your pedometer
  </Text>
</View>
`;
